pkg/hubble/relay/observer.go

```diff
@@ -32,16 +33,27 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// TODO: remove this shim once "google.golang.org/grpc" is bumped to v1.28+
```

Comment:
Because bumping this dep breaks `github.com/etcd-io/etcd` (see https://github.com/etcd-io/etcd/pull/11564) which we'd need to bump as well and as these changes need to be backported; it's a bit risky.

Reply:
I wasn't aware this was supposed to be backported. :+1: